src/odemis/gui/comp/miccanvas.py

```
@@ -274,9 +274,19 @@ def setView(self, view, tab_data):
-            self.cryotarget_fib_overlay = CryoCorrelationFibPointsOverlay(self, tab_data)
+        if (guimodel.TOOL_FIDUCIAL in tools_possible) and (guimodel.TOOL_REGION_OF_INTEREST in tools_possible):
+            self.cryotarget_fm_overlay = CryoCorrelationPointsOverlay(self, tab_data,
+                                                                      allowed_targets=(TargetType.PointOfInterest,
+                                                                                       TargetType.Fiducial))
+            self.cryotarget_fib_overlay = CryoCorrelationPointsOverlay(self, tab_data,
+                                                                       allowed_targets=(TargetType.FibFiducial,
+                                                                                        TargetType.ProjectedPOI,
+                                                                                        TargetType.ProjectedFiducial,
+                                                                                        TargetType.SurfaceFiducial))
+
+        if (guimodel.TOOL_FIDUCIAL in tools_possible) and not (guimodel.TOOL_REGION_OF_INTEREST in tools_possible):
```

Ruff (0.14.0)

Test for membership should be `not in`

Convert to `not in`

(E713)